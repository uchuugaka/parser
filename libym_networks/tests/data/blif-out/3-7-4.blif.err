"3-7-4.blif" line: 9, column: 10 [ER_MLTDEF01]: c: Defined more than once. Previsous Definition is "3-7-4.blif" line: 8, column: 9.
Error in reading 3-7-4.blif
