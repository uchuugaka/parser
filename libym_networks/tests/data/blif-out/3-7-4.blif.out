.model test1
	["3-7-4.blif" line: 3, columns:1-6]
	["3-7-4.blif" line: 3, columns:8-12]
  inputs elem: a
	["3-7-4.blif" line: 4, column: 9]
  inputs elem: b
	["3-7-4.blif" line: 4, column: 11]
  outputs elem: x
	["3-7-4.blif" line: 5, column: 10]
.names
a
	["3-7-4.blif" line: 6, column: 8]
b
	["3-7-4.blif" line: 6, column: 10]
x
	["3-7-4.blif" line: 6, column: 12]
11 1
  inputs elem: c
	["3-7-4.blif" line: 8, column: 9]
error_exit
